newdle/api.py

```diff
@@ -1,8 +1,11 @@
+import hashlib
 import uuid
 from importlib import import_module
+from urllib.parse import quote, urlencode
```

Comment:
I'd use the versions from `werkzeug.urls` (`url_quote` / `url_encode` I guess)

Reply:
Out of curiosity, why? Do they provide something more than the ones from stdlib?